File: Store/StoreAPI/Models/Product.cs
using System;
using System.Collections.Generic;

namespace StoreAPI.Models;

public partial class Product
{
    public long ProductId { get; set; }

    public long CategoryId { get; set; }

    public long? SeriesId { get; set; }

    public string Title { get; set; } = null!;

    public string? Description { get; set; }

    public decimal Cost { get; set; }

    public TimeSpan DeliveryTime { get; set; }

    public decimal Discount { get; set; }

    public virtual Category Category { get; set; } = null!;

    public virtual ICollection<OrderProduct> OrderProducts { get; set; } = new List<OrderProduct>();

    public virtual ICollection<Photo> Photos { get; set; } = new List<Photo>();

    public virtual Series? Series { get; set; }
}

File: Store/StoreTgBot/src/models/product.py
from pydantic.v1 import BaseModel
from typing import List, Optional


class RequestCreateProduct(BaseModel):
    category_id: int
    series_id: Optional[int] = None
    title: Optional[str] = None
    description: Optional[str] = None
    cost: Optional[float] = None
    delivery_time: Optional[str] = None
    discount: Optional[float] = None

    def __str__(self):
        return (
            f"category_id: {self.category_id}\n"
            f"series_id: {self.series_id}\n"
            f"title: {self.title}\n"
            f"description: {self.description}\n"
            f"cost: {self.cost}\n"
            f"delivery_time: {self.delivery_time}\n"
            f"discount: {self.discount}\n"
        )


